docs/File-Formats.md

```diff
@@ -1253,10 +1254,61 @@ The cells contain the p-value for the GSVA score: A real number, between 0.0 and
+genetic_alteration_type: TREATMENT_RESPONSE
+datatype: RESPONSE-VALUE
+stable_id: Any unique identifier using a combination of alphanumeric characters, _ and -
+profile_name: A name describing the analysis.
+profile_description: A description of the data processing done.
+data_filename: <your treatment IC50 datafile>
+show_profile_in_analysis_tab: true
+pivot_threshold_value: A threshold value beyond which a treatment response is considered effective
+value_sort_order: A flag that determines whether samples with small treatment response values are displayed first or last; can be 'ASC' for small first, 'DESC' for small last. 
+```
+
+Example:
+```
+cancer_study_identifier: study_es_0
+genetic_alteration_type: TREATMENT_RESPONSE
+datatype: RESPONSE-VALUE
+stable_id: treatment_ic50
+profile_name: IC50 values of compounds on cellular phenotype readout
+profile_description: IC50 (compound concentration resulting in half maximal inhibition) of compounds on cellular phenotype readout of cultured mutant cell lines.
+data_filename: data_treatment_ic50.txt
+show_profile_in_analysis_tab: true
+pivot_threshold_value: 0.1
+value_sort_order: ASC
+```
+
+## Note on `value_sort_order`
+When values are sorted based on the `value_sort_order`, data points at the start of the sequence are considered more significant than data points at the end. This concept is used by the oncoprint when aggregating treatment response data from multiple samples from a single patient. When `value_sort_order` is `ASC` the sample with the smallest response value will be shown for that patient. When `value_sort_order` is `DESC` the sample with the largest response value will be shown for that patient.
+
+### Treatment response IC50 data file
+The data file will be a simple tab separated format, similar to the expression data file: each sample is a column, each tested treatment (a compound or combination thereof) is a row, each cell contains treatment IC50 values for that sample x treatment combination.
+
+The first column must be named `treatment_id` and contains unique identifiers for the treatments. The `treatment_id` column can be followed by optional unique `name`, `description` and `url` columns. The other columns are sample columns; an additional column for each sample in the dataset using the sample id as the column header.
+
+The cells contain treatment response values that can be postive and negative real numbers including 0. When no response value is measured for a sample/treatment-pair the value should be "NA". Empty cell are not allowed. Numerical values van have a "<" or ">" prefix to indicate that the real value is smaller or larger than the stated response value, respectivley. Example with 3 treatments and 3 samples:
+
+<table>
+<thead><tr><th>treatment_id</th><th>name</th><th>description</th><th>url</th><th>1321N1_CENTRAL_NERVOUS_SYSTEM</th><th>22RV1_PROSTATE</th><th>42MGBA_CENTRAL_NERVOUS_SYSTEM</th></tr></thead>
```

Comment:
change `description` to `META:description`
change `treatment_id` to `stable_id`

Reply:
change `name` to `META:name`?
change `url` to `META:url`

Reply:
'META:' -tags were added to all meta fields. Validator and importer code were modified accordingly.